### PR TITLE
Accounting for raft::random detail changes

### PR DESCRIPTION
**Pull request by divyegala**
Depends on https://github.com/rapidsai/raft/pull/356/

### CODE REVIEW

**Review by rlratzel — APPROVED** [review]
just confirmed with @divyegala that the fork and version updates to `get_raft.cmake` are temporary, and will be reverted after https://github.com/rapidsai/raft/pull/356 is merged.

**Comment by divyegala** [comment]
rerun tests

**Review by ChuckHastings — APPROVED** [review]

**Comment by rlratzel** [comment]
rerun tests
This appears to be a problem related to conda installing an old cupy package. I believe the issue has since been resolved.

**Comment by rlratzel** [comment]
rerun tests

This appears to be a problem related to conda installing an old cupy package. I believe the issue has since been resolved.

**Comment by rlratzel** [comment]
rerun tests
This appears to be a problem related to conda installing an old cupy package. I believe the issue has since been resolved.

File: cpp/src/detail/shuffle_wrappers.cu [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #include <cugraph/partition_manager.hpp>
 #include <cugraph/utilities/shuffle_comm.cuh>
 
-#include <raft/random/rng.cuh>
+#include <raft/random/rng.hpp>
```
Comment (seunghwak) [inline cpp/src/detail/shuffle_wrappers.cu:21]: This one can be deleted, this file does not use raft's random number generator.

**Review by seunghwak — APPROVED** [review]
Looks good for me except for the unnecessary include statement in shuffle_wrappers.cu.

File: cpp/src/detail/shuffle_wrappers.cu (hunk 1 above)
Comment (divyegala) [reply cpp/src/detail/shuffle_wrappers.cu]: Done

**Comment by rlratzel** [comment]
rerun tests

CPU build had:
```
conda.CondaMultiError: Downloaded bytes did not match Content-Length
url: https://conda.anaconda.org/nvidia/linux-64/cudatoolkit-11.2.72-h2bc3f7f_0.tar.bz2
target_path: /opt/conda/envs/rapids/pkgs/cudatoolkit-11.2.72-h2bc3f7f_0.tar.bz2
```
and retry failed, so I wonder if the uploaded package is corrupted?

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**